tools/grift/grpc_plugins_generator.patch

```diff
@@ -0,0 +1,2417 @@
+From 0894590b5020c38106d4ebb2291994668c64f9dd Mon Sep 17 00:00:00 2001
```

Comment:
I like the idea of committing this as a patch against a well-defined version of thrift, but I would like to make sure that others agree that this is the right way to package this. For full context, these are @chedeti 's changes to the thrift code generator to make them spit out gRPC-compatible stubs and services. These will be integrated into `third_party/thrift` through `tools/grift/Dockerfile` .
Requesting an opinion from @nicolasnoble and @jtattermusch (if possible) on this particular packaging strategy.